dbt_platform_helper/utils/files.py

```diff
@@ -55,6 +57,24 @@ def generate_files_for_dir(pattern):
     generate_files_for_dir("bin/*")
 
 
+def generate_pipeline_override_files(base_path, overrides_path, output_dir, template_data={}):
```

Comment:
There's nothing inherently "pipeline" about this function, so could be renamed as maybe "generate_overrides_from_template" or something.

Reply:
Noted, will rename 👍🏻